README.md

```diff
@@ -1 +1,35 @@
-# portfolio
+
+# Portfolio: setup and mobile version skeleton
+> The goal of this project is to create the UI from the templates in Figma. Template 5 was used.
+
+![Screenshot from 2022-03-18 00-51-08](https://user-images.githubusercontent.com/90258833/158987542-c6f3fc6e-9dfe-43a1-a824-19f7e0699540.png)
+
+## Built With
+
+- html
+- css
+
+## Getting Started
+To get a local copy up and running,follow these simple steps
+
+-create a repository in your github
+-clone the repository into your local machine in the command line using git@github.com:principles31/portfolio.git
```

Comment:
Please update the command line to be `git clone git@github.com:principles31/portfolio.git`. This gives full information to the user.